.travis.yml

```diff
@@ -1,15 +1,29 @@
-language: c
-
-os:
-  - osx
+language: objective-c
 
 env:
-  - NODE_VERSION="v0.10"
-  - NODE_VERSION="v0.11"
-  - NODE_VERSION="iojs-v1.0"
+  global:
+    - secure: f4hgPmIW9fXvgomICcfe7vjJY3TFzALboVq1Nz9xrX1T25oIt7+rHd8CjPbBZvkSgSBY3zDItY5n47kAbWoFZGqmWF9K26WEnljPLab4QeLmy7GGZ7/jCTyashPucc34b5SC2SuXAMbDSSbEImguY27GQmeCnRqpDOS7kDVGZEU=
+    - secure: oSMNhXFS6Cb9GLlYaFhqABWDLYcagkz6KY5T6jFzJ6tCLhBSVbdSX9p0RhOBcIV7vZ2WW12Nbl9DTtUgccuQJWZaI504KBq2dD94R54h4hY5ZDymsRLqNZ71fD/gh3dhV9ukciHaPYPK2/N1OSxDSUAeBsZtPJxOprOH07iUNpQ=
+  matrix:
+    - NODE_VERSION="v0.10"
+    - NODE_VERSION="v0.11"
+    - NODE_VERSION="v0.12"
+    - NODE_VERSION="iojs-v1.0"
```

Comment:
If they share the same "NODE_MODULE_VERSION" represented as node-pre-gyp node_abi then it's my understanding that that implies compatibility.

https://github.com/mapbox/node-pre-gyp/blob/master/lib/util/abi_crosswalk.json#L338

The following actually may be more appropriate since the abi changed between 1.0 & 1.1 but 1.1 & 1.2 share the same abi.

NODE_VERSION="iojs-v1.0"
NODE_VERSION="io.js"